org.eclipse.xtext/src/org/eclipse/xtext/nodemodel/impl/NodeModelBuilder.java

```diff
@@ -48,6 +50,10 @@ public NodeModelBuilder() {
 		this(new GrammarElementsInterner());
 	}
 	
+	public IParseResult createParseResult(EObject model, ICompositeNode root, boolean hasErrors) {
```

Comment:
Missing since annotations here and in the following methods

Reply:
Added.